api_tests/e2e/rfc003/btc_eth/happy.js

```diff
@@ -160,11 +161,13 @@ describe("RFC003: Bitcoin for Ether", () => {
     let bob_fund_action;
 
     it("[Bob] Can get the fund action after Alice funds", async function() {
-        this.timeout(10000);
+        this.timeout(20000);
```

Comment:
❓ did you change something that made this change needed?

Reply:
No. Random debugging things.